### ISSUE TITLE
Resent crashed!

### ISSUE THREAD

**Issue by LostBro2010** [ThisIsALegitUsername/Resent-web#69]
java.lang.RuntimeException: (JavaScript) TypeError: Cannot read properties of null (reading 'vr')

TypeError: Cannot read properties of null (reading 'vr')
    at BSK (https://resclient.vercel.app/classes.js:10586:242)
    at BHU (https://resclient.vercel.app/classes.js:10588:323)
    at BhK (https://resclient.vercel.app/classes.js:15599:393)
    at $rt_array.toString [as pR] (https://resclient.vercel.app/classes.js:19529:269)
    at BV3 (https://resclient.vercel.app/classes.js:11023:271)
    at BE1 (https://resclient.vercel.app/classes.js:8384:385)
    at Bqx (https://resclient.vercel.app/classes.js:6616:518)
    at Bt8 (https://resclient.vercel.app/classes.js:3492:70)
    at BYu (https://resclient.vercel.app/classes.js:3487:183)
    at BHt (https://resclient.vercel.app/classes.js:75:155)
    at Ccu (https://resclient.vercel.app/classes.js:70:38)
    at B1_ (https://resclient.vercel.app/classes.js:67:273)
    at TeaVMThread.runner (https://resclient.vercel.app/classes.js:23:404)
    at TeaVMThread.run (https://resclient.vercel.app/classes.js:19896:111)
    at TeaVMThread.resume (https://resclient.vercel.app/classes.js:19896:9)
    at BxA.callback.bVN (https://resclient.vercel.app/classes.js:3689:310)
    at IX (https://resclient.vercel.app/classes.js:4271:24)
    at B9X (https://resclient.vercel.app/classes.js:4282:53)
    at $rt_array.toString [as Bx] (https://resclient.vercel.app/classes.js:19203:115)
    at A.AGh (https://resclient.vercel.app/classes.js:2569:121)
    at https://resclient.vercel.app/classes.js:2571:50
    at sentryWrapped (https://browser.sentry-cdn.com/6.2.2/bundle.min.js:2:53837)

eaglercraft.version = "22w43a"
eaglercraft.username = "Lost_bro"

webgl.version = WebGL 2.0 (OpenGL ES 3.0 Chromium)
webgl.renderer = Mesa DRI Intel(R) UHD Graphics 600 (GLK 2)
webgl.vendor = Intel Open Source Technology Center

webgl.anisotropicGlitch = false

window.eaglercraftOpts = {"container":"game_frame","assetsURI":"packs/blue128x.epk","serverWorkerURI":"worker_bootstrap.js","worldsFolder":"MAIN","servers":[{"serverName":"Blobcraft","serverAddress":"wss://blobcraft.minecraft.pe","hideAddress":true},{"serverName":"LegendSMP","serverAddress":"wss://legendsmp.minecraft.pe","hideAddress":true},{"serverName":"Hyper Network","serverAddress":"wss://hyper-network.me","hideAddress":true}],"relays":[{"addr":"wss://relay.deev.is/","name":"lax1dude relay #1","primary":true},{"addr":"wss://relay.lax1dude.net/","name":"lax1dude relay #2","primary":false},{"addr":"wss://relay.shhnowisnottheti.me/","name":"ayunami relay #1","primary":false}],"mainMenu":{"splashes":["Press Y to open GUI!"],"eaglerLogo":false}}

window.navigator.userAgent = Mozilla/5.0 (X11; CrOS x86_64 15183.69.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36
window.navigator.vendor = Google Inc.
window.navigator.language = en-US
window.navigator.hardwareConcurrency = 2
window.navigator.deviceMemory = 4
window.navigator.platform = Linux x86_64
window.navigator.product = Gecko

rootElement.clientWidth = 1366
rootElement.clientHeight = 617
window.innerWidth = 1366
window.innerHeight = 617
window.outerWidth = 1366
window.outerHeight = 720
window.devicePixelRatio = 1
window.screen.availWidth = 1366
window.screen.availHeight = 720
window.screen.colorDepth = 24
window.screen.pixelDepth = 24

window.location.href = https://resclient.vercel.app/#

----- Begin Minecraft Config -----
CgAAAwAFQ1BTX3kAAAAAAwAUSGVhbHRoIERpc3BsYXlfbGFzdFgAAAAAAwAUSGVhbHRoIERpc3BsYXlfbGFzdFkAAAAAAwAIa2V5Lnpvb20AAAAuAQAVTm9QYXJ0aWNsZXNfaXNFbmFibGVkAQEAD2NoYXRMaW5rc1Byb21wdAEBABRBbmltYXRpb25zX2lzRW5hYmxlZAAIACtLZXlzdHJva2VzX01vZGVTZXR0aW5nX1ByZXNzZWQgYnV0dG9uIGNvbG9yAAVCbGFjawEAJENvbWJvQ291bnRlcl9Cb29sU2V0dGluZ19UZXh0IFNoYWRvdwEDAA5IYWNrIE92ZXJsYXlfeAAAAXIDAA5IYWNrIE92ZXJsYXlfeQAAAAABABZTbW9vdGhDYW1lcmFfaXNFbmFibGVkAAEAEE5vUmFpbl9pc0VuYWJsZWQBAQAaQXV0b0dHX0Jvb2xTZXR0aW5nX09uIExvc2UABQARdm9pY2VMaXN0ZW5Wb2x1bWU/AAAAAQALc2hvd1NraW5IYXQBAwAOUmVhY2hEaXNwbGF5X3gAAAEUAQASVG9vbHRpcHNfaXNFbmFibGVkAAMADlJlYWNoRGlzcGxheV95AAAAAAMAElJlYWNoRGlzcGxheV9sYXN0WAAAAAADABJSZWFjaERpc3BsYXlfbGFzdFkAAAAAAwALa2V5LmZvcndhcmQAAAARAwAIa2V5LmxlZnQAAAAeAQAWSGFjayBPdmVybGF5X2lzRW5hYmxlZAABACZLZXlzdHJva2VzX0Jvb2xTZXR0aW5nX1JNQiBjcHMgY291bnRlcgEBABVBdXRvUmVzcGF3bl9pc0VuYWJsZWQAAwAObGltaXRGcmFtZXJhdGUAAAAAAQAUS2V5c3Ryb2tlc19pc0VuYWJsZWQAAQAUc2VlbkFja25vd2xlZGdlbWVudHMBAwAIa2V5Lmp1bXAAAAA5AQAWQ29tYm9Db3VudGVyX2lzRW5hYmxlZAABABZUb2dnbGVTcHJpbnRfaXNFbmFibGVkAAEAFlJlYWNoRGlzcGxheV9pc0VuYWJsZWQAAQALdG91Y2hzY3JlZW4AAQALaW52ZXJ0TW91c2UAAQANZmFuY3lHcmFwaGljcwABABNQb3Rpb25IVURfaXNFbmFibGVkAAEAE1dhdGVybWFya19pc0VuYWJsZWQAAQAURnVsbEJyaWdodF9pc0VuYWJsZWQAAwAFRlBTX3gAAAAACAApS2V5c3Ryb2tlc19Nb2RlU2V0dGluZ19QcmVzc2VkIHRleHQgY29sb3IABVdoaXRlAwAFRlBTX3kAAAAAAQAOSW5mb19pc0VuYWJsZWQBAwAMa2V5LmZ1bmN0aW9uAAAAIQMACUNQU19sYXN0WAAAAAADAAprZXkuc3ByaW50AAAAEwMACUNQU19sYXN0WQAAAAADAA5jaGF0VmlzaWJpbGl0eQAAAAABABBzaG93U2tpblJpZ2h0TGVnAQMADWtleS5pbnZlbnRvcnkAAAASAQASQXJtb3JIdWRfaXNFbmFibGVkAQEAGFNib2FyZCBudW1iZXJzX2lzRW5hYmxlZAABABNOb0h1cnRDYW1faXNFbmFibGVkAAMAEEtleXN0cm9rZXNfbGFzdFgAAAAABQALc2Vuc2l0aXZpdHk/AAAAAQAbQ1BTX0Jvb2xTZXR0aW5nX1RyYW5zcGFyZW50AQMAEEtleXN0cm9rZXNfbGFzdFkAAAAAAwAJcGFydGljbGVzAAAAAgMABlNsb3QgMQAAAAIDAAZTbG90IDIAAAADAwAGU2xvdCAzAAAABAMABlNsb3QgNAAAAAUDAAhDbGlja0d1aQAAABUDAAZTbG90IDUAAAAGAwAGU2xvdCA2AAAABwMABlNsb3QgNwAAAAgBABhIZWFsdGggRGlzcGxheV9pc0VuYWJsZWQBAwAGU2xvdCA4AAAACQMACmtleS5hdHRhY2v///+cAwAGU2xvdCA5AAAACgEAA2ZvZwABAA9zaG93Q29vcmRpbmF0ZXMAAQANRlBTX2lzRW5hYmxlZAEBABtDUFNfQm9vbFNldHRpbmdfVGV4dCBzaGFkb3cBAQASRnJlZUxvb2tfaXNFbmFibGVkAAMACWtleS5zbmVhawAAACoDAA9Qb3Rpb25IVURfbGFzdFgAAAAAAwAPV2F0ZXJtYXJrX2xhc3RYAAAAAAEAJktleXN0cm9rZXNfQm9vbFNldHRpbmdfTE1CIGNwcyBjb3VudGVyAQMAD1BvdGlvbkhVRF9sYXN0WQAAAAADAA9XYXRlcm1hcmtfbGFzdFkAAAAAAQAbS2V5c3Ryb2tlc19Cb29sU2V0dGluZ19KdW1wAQUAA2Zvdj6MnqYDAAhndWlTY2FsZQAAAAIFAAljaGF0U2NhbGU/gAAAAQAQaGVsZEl0ZW1Ub29sdGlwcwEBABBzaG93U2tpblJpZ2h0QXJtAQMADkFybW9ySHVkX2xhc3RYAAAAAAMADkFybW9ySHVkX2xhc3RZAAAAAAEABmNsb3VkcwAFABNjaGF0SGVpZ2h0VW5mb2N1c2VkPuMnqQgALUtleXN0cm9rZXNfTW9kZVNldHRpbmdfVW5wcmVzc2VkIGJ1dHRvbiBjb2xvcgAFQmxhY2sBABpBcm1vckh1ZF9Cb29sU2V0dGluZ19Cb290cwEDAA1hbnRpYWxpYXNNb2RlAAAAAQUAEHZvaWNlU3BlYWtWb2x1bWU/AAAAAQAUQ2xlYXIgQ2hhdF9pc0VuYWJsZWQABQALc291bmRWb2x1bWU/gAAAAQAPc2hvd1NraW5MZWZ0TGVnAQEAEWhpZGVTZXJ2ZXJBZGRyZXNzAAEADmFsbG93Rk5BV1NraW5zAAEAC3ZpZXdCb2JiaW5nAAEAEGFkdmFuY2VkVG9vbHRpcHMAAwASSGFjayBPdmVybGF5X2xhc3RYAAAAAAMADUNvb3JkaW5hdGVzX3gAAAAAAwAMS2V5c3Ryb2tlc194AAAAFAMAEkhhY2sgT3ZlcmxheV9sYXN0WQAAAAADAA1Db29yZGluYXRlc195AAAADgMADEtleXN0cm9rZXNfeQAAAAUDAAhrZXkuYmFjawAAAB8IABVDUFNfTW9kZVNldHRpbmdfQ29sb3IABVdoaXRlAwARQ29vcmRpbmF0ZXNfbGFzdFgAAAAAAwAIa2V5LmRyb3AAAAAQAQAQcGF1c2VPbkxvc3RGb2N1cwEDABFDb29yZGluYXRlc19sYXN0WQAAAAADAAxGYWtlIEhhY2tzX3gAAAMgAwALdm9pY2VQVFRLZXkAAAAvAwAMRmFrZSBIYWNrc195AAAABAMADkNvbWJvQ291bnRlcl94AAAAnQMADlRvZ2dsZVNwcmludF94AAAADwMADkNvbWJvQ291bnRlcl95AAAAAAMADlRvZ2dsZVNwcmludF95AAAACgMAEEZha2UgSGFja3NfbGFzdFgAAAAAAwAIRnJlZWxvb2sAAAAmAwAQRmFrZSBIYWNrc19sYXN0WQAAAAABACJLZXlzdHJva2VzX0Jvb2xTZXR0aW5nX1RyYW5zcGFyZW50AAMAC1BvdGlvbkhVRF94AAAAAAMAC1dhdGVybWFya194AAABLAMAC1BvdGlvbkhVRF95AAAAPAEADUNQU19pc0VuYWJsZWQAAwALV2F0ZXJtYXJrX3kAAAAEAQAJY2hhdExpbmtzAQMABkluZm9feAAAAAQBABZOb0R5bmFtaWNGT1ZfaXNFbmFibGVkAAMABkluZm9feQAAAA4BAB1Bcm1vckh1ZF9Cb29sU2V0dGluZ19MZWdnaW5ncwEBABtBcm1vckh1ZF9Cb29sU2V0dGluZ19IZWxtZXQBBQAJY2hhdFdpZHRoP4AAAAMACkFybW9ySHVkX3gAAAAAAwAKQXJtb3JIdWRfeQAAAHcBABVDb29yZGluYXRlc19pc0VuYWJsZWQACAAIbGFuZ3VhZ2UABWVuX1VTAQAPc2hvd1NraW5MZWZ0QXJtAQEAFEZha2UgSGFja3NfaXNFbmFibGVkAAEAEEF1dG9HR19pc0VuYWJsZWQAAQAZTWluaW1hbCBCb2JiaW5nX2lzRW5hYmxlZAABAAhzaG93Q2FwZQEBABJBdXRvV2Fsa19pc0VuYWJsZWQAAQAWQ2h1bmtCb3JkZXJzX2lzRW5hYmxlZAADABBIZWFsdGggRGlzcGxheV94AAABPAMAEEhlYWx0aCBEaXNwbGF5X3kAAAAAAwAHa2V5LnVzZf///50IAApsYXN0U2VydmVyAAADAAxrZXkucGlja0l0ZW3///+eAwAIa2V5LmNoYXQAAAAUAQAOc2hvd1NraW5KYWNrZXQBAwARdm9pY2VMaXN0ZW5SYWRpdXMAAAAQAwAJRlBTX2xhc3RYAAAAAAMACUZQU19sYXN0WQAAAAABAAxyZWxheVRpbWVvdXQECQAHc2VydmVycwoAAAADAQALaGlkZUFkZHJlc3MACAACaXAAH3dzczovL3BsYXkudmFuaWxsYWNyYWZ0c21wLm9yZy8IAARuYW1lABBNaW5lY3JhZnQgU2VydmVyAAEAC2hpZGVBZGRyZXNzAAgAAmlwAB13c3M6Ly9zdXMuc2hobm93aXNub3R0aGV0aS5tZQgABG5hbWUAEE1pbmVjcmFmdCBTZXJ2ZXIAAQALaGlkZUFkZHJlc3MACAACaXAAKHdzczovL2xvc3RTTVAubG9zdGJybzIwMTAucmVwbC5jby9zZXJ2ZXIIAARuYW1lABFNaW5lY3JhZnQgU2VydmVydwAFAAtjaGF0T3BhY2l0eT+AAAABABpJbmZvX0Jvb2xTZXR0aW5nX0RpcmVjdGlvbgEBABJBdXRvSnVtcF9pc0VuYWJsZWQAAQAfQXJtb3JIdWRfQm9vbFNldHRpbmdfQ2hlc3RwbGF0ZQEBAAxoaWRlSm9pbkNvZGUAAQAeS2V5c3Ryb2tlc19Cb29sU2V0dGluZ19SYWluYm93AQMACmZvcmNlV2lkdGgAAAAACAArS2V5c3Ryb2tlc19Nb2RlU2V0dGluZ19VbnByZXNzZWQgdGV4dCBjb2xvcgAFV2hpdGUDAAxjaHVua1VwZGF0ZXMAAAAAAQAWTm9Td2luZ0RlbGF5X2lzRW5hYmxlZAAFAAttdXNpY1ZvbHVtZT+AAAABACNUb2dnbGVTcHJpbnRfQm9vbFNldHRpbmdfVGV4dCBEcmF3bgEBABNGcHMgQm9vc3RfaXNFbmFibGVkAQUABWdhbW1hP4AAAAEADnNob3dPdGhlckNhcGVzAQEAC2NoYXRDb2xvdXJzAQMAEkNvbWJvQ291bnRlcl9sYXN0WAAAAAADABJUb2dnbGVTcHJpbnRfbGFzdFgAAAAAAwASQ29tYm9Db3VudGVyX2xhc3RZAAAAAAMAElRvZ2dsZVNwcmludF9sYXN0WQAAAAAIABpIaXRib3hlc19Nb2RlU2V0dGluZ19Db2xvcgADUmVkAwAMdmlld0Rpc3RhbmNlAAAAAwEACmRpZmZpY3VsdHkBAwAJa2V5LnJpZ2h0AAAAIAMACkluZm9fbGFzdFgAAAAAAwAKSW5mb19sYXN0WQAAAAABAAhhZGRlcmFsbAABABZTZWxmIE5hbWV0YWdfaXNFbmFibGVkAAMAEGFtYmllbnRPY2NsdXNpb24AAAAACAAbS2V5c3Ryb2tlc19Nb2RlU2V0dGluZ19TaXplAAVTbWFsbAEACGFuYWdseXBoAAUAEWNoYXRIZWlnaHRGb2N1c2VkP4AAAAEACmZ1bGxTY3JlZW4AAQASSGl0Ym94ZXNfaXNFbmFibGVkAQEAGUF1dG9HR19Cb29sU2V0dGluZ19PbiBXaW4BAQAUU2NvcmVib2FyZF9pc0VuYWJsZWQAAQAkQXV0b0dHX0Jvb2xTZXR0aW5nX1JlcGV0aXRpb24gYnlwYXNzAQMADmtleS5wbGF5ZXJsaXN0AAAADwMAC2ZvcmNlSGVpZ2h0AAAAAAEAGUFybW9ySHVkX0Jvb2xTZXR0aW5nX0l0ZW0BAQAcS2V5c3Ryb2tlc19Cb29sU2V0dGluZ19TbmVhawADAAVDUFNfeAAAAlYA
----- End Minecraft Config -----

window.minecraftServer = wss://lostSMP.lostbro2010.repl.co/server

**Comment by 3kh0** [comment]
Can you explain what you were doing when it crashed?

**Comment by TG20YT** [comment]
I had this issue as well. If you were playing it on the official Resent Client site (https://resclient.vercel.app/) then you just gotta refresh and it should work. I got the crash whenever I tried changing any of the video settings (Especially Buffer Anti-Aliasing). It works fine now. If you were on any other site, then try the official Resent Client site. It might crash one or two times, but then it should work.
